About.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="styles/main.css">
+  <title>Said's Portfolio</title>
+</head>
+
+<body>
+  <div class="wrapper">
+
+    <header class="site-header">
+      <img src="images/download.jpeg" alt="logo-icon" width="77" height="77">
+      <nav class="site-navigation">
+        <ul>
+          <li><a href="Home.html">HOME</a></li>
+          <li><a href="About.html">ABOUT</a></li>
+          <li><a href="Contact.html">CONTACT</a></li>
+        </ul>
+
+      </nav>
+      <h1 class="main-title">Said's Portfolio</h1>
+
+    </header>
+  </div>
+  <main>
+    <article class="main">
+      <h2>About</h2>
+      <p>About me I would describe myself as someone who is honest, caring, intelligent, hardworking, and ambitious. I have a great sense of humour. I am an easy going person and don’t get easily disturbed by down’s in my life. I am spiritual person and
+        have a good knowledge of best best healthy food and life. I also enjoy travelling, watching movies, going out for dinner, and having great intellectual conversations! Hobbies Swimming Coding Reading
+      </p>
+      <div class="">
+        <img src="https://theportfolioyas.files.wordpress.com/2014/05/myportfolio.jpg" alt="">
```

Comment:
It's always best to give an alt attribute to an `<img>`. Otherwise screen readers will completely skip it, and if the image doesn't show, nothing will be shown instead to describe it.